test/test_normalization.py

```diff
@@ -1,79 +1,79 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
-from sql_metadata import generalize_sql, remove_comments_from_sql
+from sql_metadata.parser import Parser
 
 
-def test_generalize_sql():
-    assert generalize_sql(None) is None
+def test_generalization_of_sql():
+    assert Parser(None).generalize is None
```

Comment:
```suggestion
    assert Parser("").generalize == ""
```

`None` is no longer support here.